### PR TITLE
[regression test](routine test) add case for `num_as_string`

### PR DESCRIPTION
**Pull request by liugddx**
## Proposed changes

Issue Number: close #xxx

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Comment by liugddx** [comment]
run buildall

**Comment by liugddx** [comment]
run buildall

**Comment by liugddx** [comment]
run buildall

File: be/src/clucene
Comment (HHoflittlefish777) [inline be/src/clucene:1]: remove it

**Review by sollhui — APPROVED** [review]
LGTM

**Comment by HHoflittlefish777** [comment]
run buildall

**Comment by liugddx** [comment]
run buildall

**Review by dataroaring — DISMISSED** [review]
LGTM

**Comment by liugddx** [comment]
run buildall

**Comment by liugddx** [comment]
run buildall

**Review by dataroaring — APPROVED** [review]
LGTM

**Comment by liugddx** [comment]
run feut

**Comment by liugddx** [comment]
run beut

**Comment by liugddx** [comment]
run clickbench

**Pull request merged**